cysync/src/designSystem/designComponents/customComponents/walletItem.tsx

```diff
@@ -11,11 +11,12 @@ import { useNavigate } from 'react-router-dom';
 import Routes from '../../../constants/routes';
 import {
   addressDb,
-  erc20tokenDb,
+  coinDb,
   receiveAddressDb,
+  tokenDb,
   transactionDb,
-  walletDb,
-  xpubDb
+  Wallet,
```

Comment:
pick one,, PascalCase or camelCase,  in case of interfaces, export them separately. eg: 
{module}.Types.{interfaces}

Reply:
This shall be picked up later as a cosmetic change as discussed.